index.html

```diff
@@ -14,25 +14,177 @@
+                        </ul>
+                        <div class="see_project_button"><a href="#">See project</a></div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="image_box">
+                        <img src="./images/card3.jpg" alt="">
+                    </div>
+                    <div class="text_box">
+                        <div class="project_title"><h2>Multi-Post Stories Gain+Glory</h2></div>
+                        <ul class="tags">
+                            <li><a href="#">Ruby on Rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <div class="see_project_button"><a href="#">See project</a></div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="image_box">
+                        <img src="./images/card1.jpg" alt="">
+                    </div>
+                    <div class="text_box">
+                        <div class="project_title"><h2>Multi-Post Stories Gain+Glory</h2></div>
+                        <ul class="tags">
+                            <li><a href="#">Ruby on Rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <div class="see_project_button"><a href="#">See project</a></div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="image_box">
+                        <img src="./images/card2.jpg" alt="">
+                    </div>
+                    <div class="text_box">
+                        <div class="project_title"><h2>Multi-Post Stories Gain+Glory</h2></div>
+                        <ul class="tags">
+                            <li><a href="#">Ruby on Rails</a></li>
+                            <li><a href="#">CSS</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">HTML</a></li>
+                        </ul>
+                        <div class="see_project_button"><a href="#">See project</a></div>
+                    </div>
+                </div>
+
+            </div>
+        </section>
+        <!--Recent works section end-->
+        <section class="about_me">
+            <div class="about_me_text">
+                <div class="about_primary_text">About me</div>
+                <div class="about_supporting-text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</div>
```

Comment:
- [ ] Please the title (e.g., "About myself") should use a header tag `hx`,since its better to use hx tags for headings